packages/angular_devkit/build_angular/src/webpack/configs/common.ts

```diff
@@ -457,6 +457,7 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
         }
       },
       chunkFilename: `[id]${targetInFileName}${hashFormat.chunk}.js`,
+      trustedTypes: 'webpack',
```

Comment:
NIT, I think it is better that this is added https://github.com/angular/angular-cli/blob/36b5040cc470cdb2c0554785a9a368ce3c226ad1/packages/angular_devkit/build_angular/src/webpack/configs/browser.ts#L89

Since, the common configuration is used for generating server and UT bundles where trusted types is not useful. Although adding it here, doesn't add any overhead.